src/godot.cpp

```diff
@@ -47,14 +47,15 @@ void *token = nullptr;
 
 } // namespace internal
 
-GDExtensionBinding::Callback GDExtensionBinding::core_init = nullptr;
-GDExtensionBinding::Callback GDExtensionBinding::server_init = nullptr;
-GDExtensionBinding::Callback GDExtensionBinding::scene_init = nullptr;
-GDExtensionBinding::Callback GDExtensionBinding::editor_init = nullptr;
-GDExtensionBinding::Callback GDExtensionBinding::core_terminate = nullptr;
-GDExtensionBinding::Callback GDExtensionBinding::server_terminate = nullptr;
-GDExtensionBinding::Callback GDExtensionBinding::scene_terminate = nullptr;
-GDExtensionBinding::Callback GDExtensionBinding::editor_terminate = nullptr;
+GDExtensionBinding::Callback GDExtensionBinding::init_callbacks[GDNATIVE_MAX_INITIALIZATION_LEVEL];
+GDExtensionBinding::Callback GDExtensionBinding::terminate_callbacks[GDNATIVE_MAX_INITIALIZATION_LEVEL];
+
+GDExtensionBinding::GDExtensionBinding() {
+	for (int i = 0; i < GDNATIVE_MAX_INITIALIZATION_LEVEL; i++) {
+		init_callbacks[i] = nullptr;
+		terminate_callbacks[i] = nullptr;
+	}
+}
```

Comment:
I don't think this constructor is ever called since this class is used statically.

Reply:
Hmm, what would be a good way to clear out these buffers...

Reply:
Owh I can move it into the constructor for InitObject..